File: VSTTwain/Examples/VB.NET/TwainAdvancedDemo/Dialogs/SaveSettings/JpegSaveSettingsForm.vb
Imports System.Windows.Forms

Partial Public Class JpegSaveSettingsForm
    Inherits Form
    Private _quality As Integer = 90
    Public ReadOnly Property Quality() As Integer
        Get
            Return _quality
        End Get
    End Property

    Public Sub New()
        InitializeComponent()
    End Sub

    Private Sub okButton_Click(ByVal sender As Object, ByVal e As EventArgs)
        _quality = CInt(Math.Truncate(qualityNumericUpDown.Value))
        DialogResult = DialogResult.OK
    End Sub
End Class

File: VSTTwain/Examples/CSharp/TwainAdvancedDemo/Dialogs/SaveSettings/JpegSaveSettingsForm.cs
using System;
using System.Windows.Forms;

namespace TwainAdvancedDemo
{
	public partial class JpegSaveSettingsForm : Form
	{
		int _quality = 90;
		public int Quality
		{
			get { return _quality; }
		}

		public JpegSaveSettingsForm()
		{
			InitializeComponent();
		}

		private void okButton_Click(object sender, EventArgs e)
		{
			_quality = (int)qualityNumericUpDown.Value;
			DialogResult = DialogResult.OK;
		}
	}
}